### ISSUE TITLE
Combine these websites into one backend?

### ISSUE THREAD

**Issue by dneise** [fact-project/shiftcalendar#2]
Soon these websites will play a role for the shifters during the night:

 * [Startup Checklist for the Starter](http://fact-project.org/Checklist/startup_checklist.html)
 * [Shutdown Checklist for the Parker](http://fact-project.org/Checklist/)
 * [shift calendar](http://fact-project.org/sandbox/dneise/shiftcalendar/)

Some of these websites will request the same DB tables, so some reduction might be possible, when combining them somehow.

**Comment by maxnoe** [comment]
The SH page is different from the others as it only requires read permissions, the others also require write permissions

**Comment by dneise** [comment]
right, and the SH page should not run on gate, while the others well ... should run on gate. 
I take the SH page out of the list

**Comment by dneise** [comment]
:-D Now these two comments make no sense anymore :-D
